packages/plugins/src/DocumentAssetsPlugin.ts

```diff
@@ -99,19 +97,25 @@ const DocumentAssetsPlugin: PluginType<Page, DocumentAssetsPluginOptions> = {
 
       let globbedImageDirs;
       try {
+        await fsExtra.access(resolvedSrcDir);
```

Comment:
`access` is normally discouraged. Could we just rely on glob failing?

Reply:
> Do not use fs.access() to check for the accessibility of a file before calling fs.open(), fs.readFile(), or fs.writeFile(). Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file is not accessible.